CMakeLists.txt

```diff
@@ -17,9 +17,12 @@ else()
     message("-- Configuring release build")
 endif()
 
+# Enable extra warnings to adhere to https://github.com/mapbox/cpp/issues/37
+set(DESIRED_WARNINGS "-Wall -Wextra -Wconversion -Wunreachable-code -Wuninitialized -pedantic-errors -Wold-style-cast -Wno-error=unused-variable -Wshadow -Wfloat-equal")
```

Comment:
These represent 🍇 from https://github.com/mapbox/cpp/issues/37 ?

Reply:
Yes, 🍇  and 🍊 (although I now realize I missed `-Weffc++`)